doc/development/source/tutorial-utilities.rst

```diff
@@ -92,38 +92,60 @@ displayed in the top row of the widget and also indicated in the schema.
 
 .. image:: images/warningmessage.png
 
-Messages, warning and errors are issued by methods
-`self.information(id, text)`, `self.warning(id, text)` and
-`self.error(id, text)`, respectively. A warning like the one above is
-issued by ::
+Simple messages
+...............
 
-    self.warning(42, "Data contains continuous variables. "
-                 "Discretize the data to use them.")
+If the widget only issues a single type of error, warning and/or information,
```

Comment:
This section in the doc seems a bit condensed. Should we put examples in separate rows as in the next sections?